### PR TITLE
config `cds.server.body_parser.limit` (and `cds.server` config section)

### PR DESCRIPTION
**Pull request by schwma**

### CODE REVIEW

**Comment by schwma** [comment]
Hi @danjoa, I discussed with @johannes-vogel where to document the payload limit config and he suggested that we add a new "Configuration" subsection to the `cds.server()` section to contain all `cds.server.xxx` configs (e.g. `cds.server.cors`, `cds.server.index`, `cds.server.max_request_body_size`).

What do you think, is this the right place to document the payload limit?

File: node.js/cds-server.md [hunk 1]
```diff
@@ -222,14 +222,59 @@ You can use the [served](#served) event's asynchronous nature though to wait for
...
+
+
+
+### Toggle Default Index Page
+
+The default index page is not served if `NODE_ENV` is set to `production`. Set `cds.server.index = true` to restore the index page in production.
+
+
+
+### Maximum Request Body Size
```
Comment (schwma) [inline node.js/cds-server.md:250]: Note: section name is still "Maximum Request Body Size", but flag name is `cds.server.body_parser.limit`.

However, express also explains that its `limit` config "Controls the maximum request body size.": https://expressjs.com/en/api.html#express.json

File: node.js/cds-server.md [hunk 2]
```diff
@@ -222,14 +222,59 @@ You can use the [served](#served) event's asynchronous nature though to wait for
...
 })
 ```
 
 
+
+## Configuration
+
+The behavior of the built-in server can be customized through the options documented in the following sections.
+
+### CORS Middleware
```
Comment (johannes-vogel) [inline node.js/cds-server.md:238]: link to https://pages.github.tools.sap/cap/docs/node.js/best-practices#cross-origin-resource-sharing-cors

File: node.js/cds-server.md [hunk 3]
```diff
@@ -222,14 +222,59 @@ You can use the [served](#served) event's asynchronous nature though to wait for
...
+
+The built-in CORS middleware can be enabled explicitly with `cds.server.cors = true`.  By default, this is `false` if in production.
+
+
+
+### Toggle Default Index Page
```
Comment (johannes-vogel) [inline node.js/cds-server.md:244]: if we have described the index page somewhere, I'd link to it from here as well.
With the links users will understand easier what they toggle/configure.

Comment (schwma) [reply node.js/cds-server.md:244]: The index page is mentioned and shown as a screenshot in the getting started section: https://cap.cloud.sap/docs/get-started/in-a-nutshell#generic-index-html

**Review by smahati — APPROVED** [review]

**Pull request merged**